assets/scripts/info_bubble/WidthControl.jsx

```diff
@@ -129,10 +129,9 @@ function mapStateToProps (state, ownProps) {
     locale: state.locale.locale
   }
 }
-function mapDispatchToProps (dispatch, ownProps) {
-  return {
-    incrementSegmentWidth: (add, precise, value) => dispatch(incrementSegmentWidth(ownProps.position, add, precise, value))
-  }
+
+const mapDispatchToProps = {
+  incrementSegmentWidth
```

Comment:
Hmm, it looks like this is removing `position` from the call to `incrementSegmentWidth`, is that on purpose?